### DISCUSSION TITLE
"Repair all" in Settings

### DISCUSSION THREAD

**Discussion by steinbeck** [home-assistant/frontend#20710] [Other feature requests]
I regularly get "repair" messages in my settings (I assume that this is the english term - I use a German user interface) that my 10+ Shelly's push update has failed (see screenshot). An "ignore all", or so, function would be extremely handy. At the moment, I need to go each and everyone of them and ignore. 
![push update repair](https://github.com/home-assistant/frontend/assets/107318/ab894052-4a39-48aa-b9fc-a40e305ee746)

**Comment by ildar170975** [comment]
https://github.com/home-assistant/frontend/discussions/18634
